core/lib/multivm/src/versions/vm_latest/vm.rs

```diff
@@ -79,6 +79,23 @@ impl<S: WriteStorage, H: HistoryMode> Vm<S, H> {
         self.state.local_state.callstack.current.ergs_remaining
     }
 
+    pub(crate) fn ask_decommitter(&self, hashes: Vec<H256>) -> Vec<Vec<u8>> {
+        let mut result = vec![];
+        for hash in hashes {
```

Comment:
Nit: `hashes` aren't really consumed, so you can pass `&[H256]`. Also, since bytecode hashes correspond 1-to-1 to resolved bytecodes, it's possbile to use `hashes.iter().map(..).collect()` workflow.